File: hw/nase/u3/hw24CompositePattern/CompossitePattern.javaScript/model/Teller.js
import { Employee } from './Employee.js';

//Jennyfer Nase

export class Teller extends Employee {
    constructor(aName = null) {
        super();
        this.title = "Teller";
        if (aName !== null) {
            this.name = aName;
        }
    }

    stateName() {
        super.stateName();
    }
}

File: hw/nase/u3/hw24CompositePattern/CompossitePattern.java/Employee/src/main/java/espe/edu/ec/employee/model/Teller.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package espe.edu.ec.employee.model;

/**
 *
 * @author Jennyfer Nase
 */
public class Teller extends Employee {

    public Teller(String aName) {
        this();
        this.name = aName;
    }

    public Teller() {
        this.title = "Teller";
    }

    @Override
    public void stateName() {
        super.stateName();
    }
}